src/EntityComponentManager.cc

```diff
@@ -199,6 +211,7 @@ void EntityComponentManager::ClearNewlyCreatedEntities()
 {
   std::lock_guard<std::mutex> lock(this->dataPtr->entityCreatedMutex);
   this->dataPtr->newlyCreatedEntities.clear();
+  this->dataPtr->modifiedComponents.clear();
```

Comment:
should this call be in `ClearNewlyCreatedEntities` or maybe it fits better in `SetAllComponentsUnchanged`?

Reply:
Moved to `SetAllComponentsUnchanged` 70c432050ff7c18d5cb3f79de5cb697ce950ff14